### PR TITLE
[Nim] Add Basolato #2691

### LINKED ISSUE #2691: [Nim] Add basolato
Hi @itsumura-h,

Thanks for your approval in https://github.com/itsumura-h/nim-basolato/issues/73. The scenario `here` is very simple (yet) :

+ `GET` `/`, **SHOULD** return a successful status code and an empty string
+ `GET` `/user/:id`, **SHOULD** return a successful status code and the id
+ `POST` `/user`, **SHOULD** return a successful status code and an empty string

Feel free to `PR`

Regards,

### PR DESCRIPTION
**Pull request by itsumura-h**
close #2691

And I updated other Nim frameworks to run all Nim project successfully.

### CODE REVIEW

File: nim/basolato/server.nimble [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+license       = "MIT"
+srcDir        = "."
+bin           = @["server"]
+
+backend       = "c"
+
+# Dependencies
+
+requires "nim >= 1.2.4"
+requires "https://github.com/itsumura-h/nim-basolato >= 0.5.1"
```
Comment (waghanza) [inline nim/basolato/server.nimble:15]: Is there not a **published** version of `basolato` ?

File: nim/basolato/server.nimble [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+requires "httpbeast >= 0.2.2"
+requires "cligen >= 0.9.41"
+requires "templates >= 0.5"
+requires "bcrypt >= 0.2.1"
+requires "nimAES >= 0.1.2"
+requires "https://github.com/enthus1ast/flatdb >= 0.2.4"
```
Comment (waghanza) [inline nim/basolato/server.nimble:21]: Is there not a **published** version of `flatdb` ?

File: nim/basolato/server.nimble (hunk 1 above)
Comment (itsumura-h) [reply nim/basolato/server.nimble:15]: Basolato is still develop version and not published to nimble package yet.

File: nim/basolato/server.nimble (hunk 2 above)
Comment (itsumura-h) [reply nim/basolato/server.nimble]: fixed

File: nim/basolato/public/basolato.svg [hunk 3]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (waghanza) [inline nim/basolato/public/basolato.svg:1]: is this file needed ?

Comment (itsumura-h) [reply nim/basolato/public/basolato.svg:1]: Actuary it's not necessary to run app. Should I remove all files which are not called and make project minimal?

Comment (waghanza) [reply nim/basolato/public/basolato.svg:1]: Yes please

Comment (waghanza) [reply nim/basolato/public/basolato.svg:1]: @itsumura-h any time for this ?

**Comment by itsumura-h** [comment]
@waghanza 
I made the project minimal. Please check it out!
